src/crowdsale/StakedVestedCrowdSale.sol

```diff
@@ -46,18 +45,24 @@ contract StakedVestedCrowdSale is VestedCrowdSale {
     }
 
     function startSale(Sale memory sale, StakingConfig memory stakingConfig, uint256 cliff, uint256 duration) public returns (uint256 saleId) {
+        if (IERC20Metadata(address(stakingConfig.stakedToken)).decimals() != 18) {
+            revert BadDecimals();
+        }
         saleId = super.startSale(sale, cliff, duration);
         salesStaking[saleId] = stakingConfig;
     }
 
-    function startSale(Sale memory sale, IERC20 stakedToken, TokenVesting stakesVesting, uint256 initialPrice, uint256 cliff, uint256 duration)
-        public
-        returns (uint256 saleId)
-    {
-        //1 DAO = 1 bid, current price
-        uint256 wadDaoInBiddingPrice = 1 * FP.WAD;
-
-        return startSale(sale, StakingConfig(stakedToken, stakesVesting, initialPrice, wadDaoInBiddingPrice, 0), cliff, duration);
+    function startSale(
+        Sale memory sale,
+        IERC20 stakedToken,
+        TokenVesting stakesVesting,
+        uint256 wadFixedDaoPerBidPrice,
```

Comment:
I guess this means we're set on doing a fixed price auctionToken/daoToken crowdsale?

Reply:
right now, yes.